enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -275,6 +275,89 @@ describeWithToken("formatting > sandboxes", () => {
       });
     });
 
+    it.skip("should allow joins to the sandboxed table (metabase-enterprise#154)", () => {
+      const COLLECTION_GROUP_ID = 4;
+      withSampleDataset(
+        ({ PEOPLE, PEOPLE_ID, ORDERS_ID, PRODUCTS_ID, REVIEWS_ID }) => {
+          cy.log(
+            "**-- 1. Sandbox `People` table on `user_id` attribute for `data` group --**",
+          );
+
+          cy.request("POST", "/api/mt/gtap", {
+            attribute_remappings: {
+              user_id: ["dimension", ["field-id", PEOPLE.ID]],
+            },
+            card_id: null,
+            group_id: COLLECTION_GROUP_ID,
+            table_id: PEOPLE_ID,
+          });
+
+          cy.log("**-- 2. Fetch permissions graph --**");
+
+          cy.request("GET", "/api/permissions/graph", {}).then(
+            ({ body: { groups, revision } }) => {
+              // Update permissions for `collections` group [id: 4]
+              // This mutates the original `groups` object => we'll pass it next to the `PUT` request
+              groups[COLLECTION_GROUP_ID] = {
+                1: {
+                  schemas: {
+                    PUBLIC: {
+                      [ORDERS_ID]: "all",
+                      [PEOPLE_ID]: { query: "segmented", read: "all" },
+                      [PRODUCTS_ID]: "all",
+                      [REVIEWS_ID]: "all",
+                    },
+                  },
+                },
+              };
+
+              cy.log("**-- 3. Update/save permissions --**");
+
+              cy.request("PUT", "/api/permissions/graph", {
+                groups,
+                revision,
+              });
+            },
+          );
+        },
+      );
+
+      signOut();
+
+      cy.log("**-- Logging in as sandboxed user --**");
+      cy.request("POST", "/api/session", {
+        username: sandboxed_user.email,
+        password: sandboxed_user.password,
+      });
+
+      // Go straight to orders table in custom questions
+      cy.visit("/question/new?database=1&table=2&mode=notebook");
```

Comment:
This will be abstracted into a global helper function anyway. It will use variables instead of hard coded values.

Reply:
My point is that it's much easier to read variables instead of guessing what table ID 2 is supposed to be.

Reply:
Absolutely agreed!
I'm just saying that we're probably a long overdue abstracting this and making it global. It didn't make sense that I change it here in one place only when I plan to use helper function instead.

But you have the point and it will be addressed.